components/clamav/clamav-core.p5m

```diff
@@ -0,0 +1,196 @@
+hardlink path=usr/bin/sigtool target=../lib/isaexec pkg.linted=true
+hardlink path=usr/bin/clambc target=../lib/isaexec pkg.linted=true
+hardlink path=usr/bin/clamdscan target=../lib/isaexec pkg.linted=true
+hardlink path=usr/bin/freshclam target=../lib/isaexec pkg.linted=true
+hardlink path=usr/sbin/clamd target=../lib/isaexec pkg.linted=true
+
+file path=usr/lib/libclamunrar_iface.la mode=0555 owner=root group=sys
+file path=usr/lib/libclamav.la mode=0555 owner=root group=sys
+file path=usr/lib/libclamunrar.la mode=0555 owner=root group=sys
+file path=usr/lib/pkgconfig/libclamav.pc mode=0444 owner=root group=sys
+
+file path=usr/lib/libclamunrar_iface.so.$(COMPONENT_VERSION_LIB_FULL) mode=0555 owner=root group=sys
+link path=usr/lib/libclamav.so.$(COMPONENT_VERSION_LIB_MAJOR) target=libclamav.so.$(COMPONENT_VERSION_LIB_FULL)
+link path=usr/lib/libclamunrar_iface.so target=libclamunrar_iface.so.$(COMPONENT_VERSION_LIB_FULL)
+file path=usr/lib/libclamunrar.so.$(COMPONENT_VERSION_LIB_FULL) mode=0555 owner=root group=sys
+link path=usr/lib/libclamunrar.so target=libclamunrar.so.$(COMPONENT_VERSION_LIB_FULL)
+file path=usr/lib/libclamav.so.$(COMPONENT_VERSION_LIB_FULL) mode=0555 owner=root group=sys
+link path=usr/lib/libclamunrar_iface.so.$(COMPONENT_VERSION_LIB_MAJOR) target=libclamunrar_iface.so.$(COMPONENT_VERSION_LIB_FULL)
+link path=usr/lib/libclamav.so target=libclamav.so.$(COMPONENT_VERSION_LIB_FULL)
+link path=usr/lib/libclamunrar.so.$(COMPONENT_VERSION_LIB_MAJOR) target=libclamunrar.so.$(COMPONENT_VERSION_LIB_FULL)
+
+file path=usr/lib/$(MACH64)/libclamunrar_iface.la mode=0555 owner=root group=sys
+file path=usr/lib/$(MACH64)/libclamav.la mode=0555 owner=root group=sys
+file path=usr/lib/$(MACH64)/libclamunrar.la mode=0555 owner=root group=sys
+file path=usr/lib/$(MACH64)/pkgconfig/libclamav.pc mode=0444 owner=root group=sys
+
+file path=usr/lib/$(MACH64)/libclamunrar_iface.so.$(COMPONENT_VERSION_LIB_FULL) mode=0555 owner=root group=sys
+link path=usr/lib/$(MACH64)/libclamav.so.$(COMPONENT_VERSION_LIB_MAJOR) target=libclamav.so.$(COMPONENT_VERSION_LIB_FULL)
+link path=usr/lib/$(MACH64)/libclamunrar_iface.so target=libclamunrar_iface.so.$(COMPONENT_VERSION_LIB_FULL)
+file path=usr/lib/$(MACH64)/libclamunrar.so.$(COMPONENT_VERSION_LIB_FULL) mode=0555 owner=root group=sys
+link path=usr/lib/$(MACH64)/libclamunrar.so target=libclamunrar.so.$(COMPONENT_VERSION_LIB_FULL)
+file path=usr/lib/$(MACH64)/libclamav.so.$(COMPONENT_VERSION_LIB_FULL) mode=0555 owner=root group=sys
+link path=usr/lib/$(MACH64)/libclamunrar_iface.so.$(COMPONENT_VERSION_LIB_MAJOR) target=libclamunrar_iface.so.$(COMPONENT_VERSION_LIB_FULL)
+link path=usr/lib/$(MACH64)/libclamav.so target=libclamav.so.$(COMPONENT_VERSION_LIB_FULL)
+link path=usr/lib/$(MACH64)/libclamunrar.so.$(COMPONENT_VERSION_LIB_MAJOR) target=libclamunrar.so.$(COMPONENT_VERSION_LIB_FULL)
+
+### If --enable-libltdl-install, conflicts with libtool
```

Comment:
ok. again, initially built with this like in-house, then discovered that the OS has the library ;)